udtube/modules.py

```diff
@@ -16,6 +16,10 @@
 from . import data, defaults, encoders
 
 
+class Error(Exception):
+    pass
```

Comment:
Why not use a better Error class name? - `MisconfigurationError`

Reply:
pure YAGNI, you can add more depth in the hierarchy later if needed but unlikely because how would you recover from this error anyways?

Reply:
Fair enough, but I like the `MisconfigurationError` idea as it can be used in a bunch of different places, but different PR.

Reply:
SGTM for later. I would actually define `udtube.Error` and then inherit from that. Until I'm ready to do that though I usually just do module-level errors since knowing which module the error happened in is extremely useful information.